packages/teleterm/src/ui/services/clusters/clustersService.ts

```diff
@@ -72,7 +72,6 @@ export class ClustersService extends ImmutableStore<ClustersServiceState> {
   async logout(clusterUri: string) {
     // TODO(gzdunek): logout and removeCluster should be combined into a single acton in tshd
     await this.client.logout(clusterUri);
-    await this.syncClusterInfo(clusterUri);
```

Comment:
Why would we need to try and `syncClusterInfo` _after_ logging out? I removed it because now that `GetCluster` has a `addMetadataToRetryableError`, logging out would fail here. "no SSH auth methods found. are you logged in?". Of course we aren't we just logged out the line above!  I think this line might have just been an error from the start (git blame says back in febuary) and it wasn't never noticed since it was essentially ignoring the error.

Reply:
Just for posterity, I'm linking my comment from another PR https://github.com/gravitational/teleport/pull/17497#discussion_r998037506 which explains that this is probably a remnant of times when logging out and removing a cluster from the list were two separate actions.